### PR TITLE
fix(css/parser): lexer errors

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Add lexer errors for CSS parser.

We need to sort them in test because when we consume using tokens we run the lexer firstly and then the parser, but when we parser using string we lazy consume tokens, so errors have different order.

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_parser/src/lexer/mod.rs [hunk 1]
```diff
@@ -198,7 +198,15 @@ where
         self.input.reset_to(self.cur_pos);
     }
 
-    fn read_token(&mut self) -> LexResult<Token> {
+    #[inline(always)]
```
Comment (kdy1) [inline crates/swc_css_parser/src/lexer/mod.rs:201]: Can you remove this and instead add `#[cold]`?

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thank you!
`#[inline(always)]` makes debug build slower, and for error reporting functions, `#[cold]` is better

**Comment by alexander-akait** [comment]
@kdy1 Fixed, also fixed `for swc_html_parser`

**Review by kdy1 — APPROVED** [review]
Thank you!

---

swc-bump:
 - swc_css_parser

**Pull request merged**